src/partials/languages/he.html

```diff
@@ -24,22 +24,42 @@
 {% macro t(key) %}{{ {
   "language": "he",
   "direction": "rtl",
-  "clipboard.copy": "העתק ללוח",
+  "announce.dismiss": "לא להציג את זה שוב",
+  "clipboard.copy": "העתקה ללוח",
   "clipboard.copied": "הועתק ללוח",
-  "edit.link.title": "ערוך דף זה",
-  "footer.previous": "קודם",
-  "footer.next": "הַבָּא",
+  "consent.accept": "לקבל",
+  "consent.manage": "לנהל הגדרות",
+  "consent.reject": "לדחות",
+  "edit.link.title": "עריכת הדף הזה",
+  "footer.previous": "הקודם",
+  "footer.next": "הבא",
+  "footer.title": "כותרת תחתונה",
+  "header.title": "כותרת עליונה",
   "meta.comments": "הערות",
-  "meta.source": "מָקוֹר",
+  "meta.source": "מקור",
+  "nav.title": "ניווט",
+  "search.config.lang": "he",
   "search.config.pipeline": " ",
-  "search.placeholder": "לחפש",
-  "search.result.placeholder": "הקלד כדי להתחיל לחפש",
+  "search.config.separator": "[\\s\\-]+",
```

Comment:
The separator is only necessary to provide if it deviates from the standard one defined for `en` which it doesn't. Please remove this line. I'm going to improve the documentation for providing and updating translations in the near future.

Reply:
Thanks for the comment. Done. I actually had a suspicion about it.